srcpkgs/gmsh/template

```diff
@@ -0,0 +1,44 @@
+# Template file for 'gmsh'
+pkgname=gmsh
+version=4.0.7
+revision=1
+wrksrc=${pkgname}-${version}-source
+build_style=cmake
+configure_args="
+ -DENABLE_BUILD_SHARED=1
+ $(vopt_if numpy '-DENABLE_NUMPY=1' '-DENABLE_NUMPY=0')
```

Comment:
you can do `-DENABLE_NUMPY=$(vopt_if numpy 'ON' 'OFF')`

Reply:
Ah thanks, didn't know that